### PR TITLE
Apply redwood theme

### PR DESCRIPTION
**Pull request by cm9361**
### Summary

**What was changed:**
The Terra Badge, Base, Card, Divider, Hyperlink, Image, Overlay, and Signature components were updated to support the Redwood theme

**Why it was changed:**
The change was made to start the rebranding of our solutions to match Oracle standards.

### Testing

This change was tested using:

- [X] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-10149 
UXPLATFORM-10031
UXPLATFORM-10011
UXPLATFORM-10009
UXPLATFORM-10003
UXPLATFORM-9987
UXPLATFORM-9950

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-hyperlink/tests/wdio/__snapshots__/reference/redwood-theme/en/chrome_tiny/hyperlink-spec/hyperlink-button.png
Comment (adoroshk) [inline packages/terra-hyperlink/tests/wdio/__snapshots__/reference/redwood-theme/en/chrome_tiny/hyperlink-spec/hyperlink-button.png:1]: I am not sure why it looks like there is an example missing on the top line of this png?

File: packages/terra-overlay/tests/wdio/__snapshots__/reference/redwood-theme/en/chrome_huge/overlay-spec/default.png
Comment (adoroshk) [inline packages/terra-overlay/tests/wdio/__snapshots__/reference/redwood-theme/en/chrome_huge/overlay-spec/default.png:1]: Just double-checking to ensure that overlay looks as expected and should not have any text?

**Review by adoroshk — APPROVED** [review]
I left 2 questions to double-check, otherwise LGTM

File: packages/terra-overlay/tests/wdio/__snapshots__/reference/redwood-theme/en/chrome_huge/overlay-spec/default.png
Comment (cm9361) [reply packages/terra-overlay/tests/wdio/__snapshots__/reference/redwood-theme/en/chrome_huge/overlay-spec/default.png:1]: Great question.  This path shows that text should not be present: raw/tests/cerner-terra-core-docs/overlay/overlay/default-overlay

File: packages/terra-hyperlink/tests/wdio/__snapshots__/reference/redwood-theme/en/chrome_tiny/hyperlink-spec/hyperlink-button.png
Comment (cm9361) [reply packages/terra-hyperlink/tests/wdio/__snapshots__/reference/redwood-theme/en/chrome_tiny/hyperlink-spec/hyperlink-button.png:1]: There is a link missing.  However, this is the same behavior seen in the existing tests.  We might need to reevaluate the unit test and how the snapshot is captured.  I can log a JIRA, if desired.

**Review by artpark — APPROVED** [review]

**Review by sycombs — APPROVED** [review]

**Review by sdadn — APPROVED** [review]

**Pull request merged**